en/micro-integrator/docs/references/connectors/salesforce-rest-connector/sf-rest-inbound-endpoint-example.md

```diff
@@ -0,0 +1,123 @@
+
+2. Right click on **Created Integration Project** -> **New** -> **Inbound Endpoint** -> **Create A New Inbound Endpoint** -> **Inbound Endpoint Creation Type**and select as **custom** -> Click **Next**.
+
+   <img src="../../../../assets/img/connectors/salesforce-inboundEP-create-new-IE.png" title="Creating inbound endpoint" width="400" alt="Creating inbound endpoint" style="border:1px solid black"/>
+
+3. Click on **Inbound Endpoint** in design view and under `properties` tab, update class name to `org.wso2.carbon.inbound.salesforce.poll.SalesforceStreamData`. 
+
+4. Navigate to the source view and update it with the following configuration as required. 
+
+   ```xml
+   <?xml version="1.0" encoding="UTF-8"?>
+   <inboundEndpoint xmlns="http://ws.apache.org/ns/synapse"
+                    name="SaleforceInboundEP"
+                    sequence="test"
+                    onError="fault"
+                    class="org.wso2.carbon.inbound.salesforce.poll.SalesforceStreamData"
+                    suspend="false">
+      <parameters>
+         <parameter name="inbound.behavior">polling</parameter>
+         <parameter name="interval">100</parameter>
+         <parameter name="sequential">true</parameter>
+         <parameter name="coordination">true</parameter>
+         <parameter name="connection.salesforce.replay">false</parameter>
+         <parameter name="connection.salesforce.EventIDStoredFilePath">/home/kasun/Documents/SalesForceConnector/a.txt</parameter>
+         <parameter name="connection.salesforce.packageVersion">37.0</parameter>
+         <parameter name="connection.salesforce.salesforceObject">/topic/Account</parameter>
+         <parameter name="connection.salesforce.loginEndpoint">https://login.salesforce.com</parameter>
+         <parameter name="connection.salesforce.userName">Username</parameter>
+         <parameter name="connection.salesforce.password">test123XXXXXXXXXX</parameter>
+         <parameter name="connection.salesforce.waitTime">5000</parameter>
+         <parameter name="connection.salesforce.connectionTimeout">20000</parameter>
+         <parameter name="connection.salesforce.soapApiVersion">22.0</parameter>
+      </parameters>
+   </inboundEndpoint>
+   ```
+   Sequence to process the message.
+   
+   In this example for simplicity we will just log the message, but in a real world use case, this can be any type of message mediation.
+ 
+   ```xml
+   <?xml version="1.0" encoding="UTF-8"?>
+   <sequence name="test" onError="fault" xmlns="http://ws.apache.org/ns/synapse">
+       <log level="full"/>
+       <drop/>
+   </sequence>
+   ```
+> **Note**: To configure the `connection.salesforce.password` parameter value please use the steps given under topic `Reset Security Token` in the [sf-rest inbound endpoint configuration](sf-rest inbound endpoint configuration.md) document.
```

Comment:
```suggestion
> **Note**: To configure the `connection.salesforce.password` parameter value, please use the steps given under the topic `Reset Security Token` in the [sf-rest inbound endpoint configuration](sf-rest inbound endpoint configuration.md) document.
```

Does this link work? File name normally doesn't have spaces.

Reply:
Updated.